code/datums/spells/conjure.dm

```diff
@@ -50,15 +50,11 @@
 
 	return
 
-/obj/effect/proc_holder/spell/aoe_turf/conjure/summonEdSwarm //test purposes
+/obj/effect/proc_holder/spell/aoe/turf/conjure/summonEdSwarm //test purposes
 	name = "Dispense Wizard Justice"
 	desc = "This spell dispenses wizard justice."
 
 	summon_type = list(/mob/living/simple_animal/bot/ed209)
 	summon_amt = 10
 	newVars = list("emagged" = 1,"name" = "Wizard's Justicebot")
-
-/obj/effect/proc_holder/spell/aoe_turf/conjure/summonEdSwarm/create_new_targeting()
```

Comment:
I prefer to have the targeting part of a spell not be part of the path of the spell and instead have each spell implement its own targeting. The original code still violated this but this was due to me lacking the time to properly convert the `aoe_turf` spells to their own type. Sorry for that 😅 

Having spell typepaths with their targeting in the path will make the system rigid. Want a conjure which is not AOE based? Well tough luck since the conjure spell is an aoe by path define.
Basically, the spell path should be what the spell does with it's targets. Then you can have subtypes on that which implement different targeting systems.

In this case, to avoid a big spell refactor, it would work to have `/spell/aoe/conjure`. Then have the actual subtypes implement their own `spell_targeting` object. Later somebody can remove the `aoe` part of the path and just have them implement the logging logic themselves... or maybe even have a logging_handler object as a composition instead of inheriting it.